hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -130,8 +137,6 @@
   protected final ConcurrentHashMap<String, ReplicationSourceShipper> workerThreads =
       new ConcurrentHashMap<>();
 
-  private AtomicLong totalBufferUsed;
```

Comment:
Since we are cleaning out this extra, unnecessary variable, could we make "this.manager" package private (or provide a package private getter), so that source readers and source shippers don't have to define an additional reference to ReplicationSourceManager?

Reply:
Good idea. Let me refator it. And after I rethink about this, the ReplicationSourceManager is still needed by ReplicationSource. But only need the getTotalBuffered/getTotalLimit/getGlobalMetrics now. Maybe introduce a new interface in the future.